src/main/java/duke/Todo.java

```diff
@@ -0,0 +1,23 @@
+package duke;
+
+public class Todo extends Task {
+
+    public Todo(String description) {
+        super(description);
+    }
+
+    public Todo(String description, String status) {
+        super(description, status);
+    }
+
+    @Override
+    public String toString() {
+        return "[T]" + super.toString();
+    }
+
+    @Override
+    public String toData() {
+        return "T|" + super.toData();
+    }
+
+}
```

Comment:
I think the Todo.java, Event.java, Task.java, and Deadline.java can be classified under a package name Tasks.